### PR TITLE
Feature/disable zaken in use

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/GemeenteUtrecht/record-management-app/issues/37

### CODE REVIEW

File: src/rma/conf/base.py [hunk 1]
```diff
@@ -368,3 +368,4 @@
 CELERY_TASK_SOFT_TIME_LIMIT = 600
 
 ZAKEN_PER_TASK = 10
+ZAKEN_PER_QUERY = 1000000
```
Comment (sergei-maertens) [inline src/rma/conf/base.py:371]: A cool Python 3.6 feature is that you can use underscores in literals to improve readability:

```suggestion
ZAKEN_PER_QUERY = 1_000_000
```

File: src/rma/js/forms/inputs.js [hunk 2]
```diff
@@ -5,16 +5,15 @@ import { Label } from './label';
...
+const Input = ({ type='text', id='', name='', initial, classes=null, checked=false, onBlur, onChange, required=false, disabled=false }) => {
     return (
         <input
             name={name}
             type={type}
             id={id}
             defaultValue={initial || ''}
             checked={checked}
-            className={classNames}
+            className={classes}
```
Comment (sergei-maertens) [inline src/rma/js/forms/inputs.js:16]: why was this change reverted?

Comment (annashamray) [reply src/rma/js/forms/inputs.js:16]: there is no `classNames` variable in this component. I assumed that `classes` props var should be used here

Comment (sergei-maertens) [reply src/rma/js/forms/inputs.js:16]: line 9:

```js
const classNames = classes ??`input__control input__control--${type}`;
```

Comment (annashamray) [reply src/rma/js/forms/inputs.js:16]: Oh, sorry, I accidentally removed this line than I did rebase. I'll fix it

Comment (annashamray) [reply src/rma/js/forms/inputs.js]: fixed

**Pull request merged**